builds/e2e/templates/e2e-setup.yaml

```diff
@@ -22,35 +22,14 @@ steps:
       TestRootCaPassword,
       TestBlobStoreSas
 
-- pwsh: |
-    $imageBuildId = $(resources.pipeline.images.runID)
-    $packageBuildId = $(resources.pipeline.packages.runID)
-    if ('$(az.pipeline.images.buildId)')
-    {
-      Write-Output '>> User supplied az.pipeline.images.buildId=$(az.pipeline.images.buildId)'
-      $imageBuildId = '$(az.pipeline.images.buildId)'
-    }
-    if ('$(az.pipeline.packages.buildId)')
-    {
-      Write-Output '>> User supplied az.pipeline.packages.buildId=$(az.pipeline.packages.buildId)'
-      $packageBuildId = '$(az.pipeline.packages.buildId)'
-    }
-
-    Write-Output "##vso[task.setvariable variable=imageBuildId]$imageBuildId"
-    Write-Output "##vso[task.setvariable variable=packageBuildId]$packageBuildId"
-
-    Write-Output ">> Package Build ID=$packageBuildId"
-    Write-Output ">> Image Build ID=$imageBuildId"
-  displayName: Override artifacts with user-supplied args
-
 - task: DownloadBuildArtifacts@0
   displayName: Get Docker image info
   inputs:
     buildType: specific
     project: $(resources.pipeline.images.projectID)
     pipeline: $(resources.pipeline.images.pipelineName)
     buildVersionToDownload: specific
-    buildId: $(imageBuildId)
+    buildId: $(resources.pipeline.images.runID)
```

Comment:
You also need to replace `$(packageBuildId)` in the next task below with `$(resources.pipeline.packages.runID)`.

Reply:
Ah that's why there were conflicts with the cherry pick. Thought it was some git weirdness.